docs/user/managing-devices.md

```diff
@@ -119,8 +155,8 @@ As good practice, labels should take the form of `key=value` pairs, whereby the
 
 Labels must follow certain rules to be valid (in fact, these are the same as for [Kubernetes](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)):
 
-* Keys and value must each be 63 characters or less. Value may be omitted.
-* Keys and values may consist of alphanumeric characters (`a-z`, `A-Z`, `0-9`). They may also contain dashes (`-`), underscores (`_`), dots (`.`), but not as the first or last character.
+- Keys and value must each be 63 characters or less. Value may be omitted.
+- Keys and values may consist of alphanumeric characters (`a-z`, `A-Z`, `0-9`). They may also contain dashes (`-`), underscores (`_`), dots (`.`), but not as the first or last character.
```

Comment:
We're using `*` for unordered lists everywhere in the docs.

Reply:
the linter was failing on that was the only reason I changed this

Reply:
strange maybe it was a warning fine now